modules/message-lane/src/lib.rs

```diff
@@ -136,6 +134,66 @@ pub trait Config<I = DefaultInstance>: frame_system::Config {
 	type MessageDispatch: MessageDispatch<Self::InboundMessageFee, DispatchPayload = Self::InboundPayload>;
 }
 
+/// Extended weight info.
+pub trait WeightInfoEx: WeightInfo {
```

Comment:
```suggestion
pub trait WeightInfoExt: WeightInfo {
```

I think the convention is to use `Ext` for _Extension_ traits (`FutureExt` and `StreamExt` come to mind)